pkg/doc/verifiable/linked_data_proof_test.go

```diff
@@ -41,7 +46,107 @@ func Test_keyResolverAdapter_Resolve(t *testing.T) {
 		kra := &keyResolverAdapter{pubKeyFetcher: SingleKey(privateKey.Public())}
 		resolvedPubKey, err := kra.Resolve("any")
 		require.Error(t, err)
-		require.EqualError(t, err, "expecting []byte public key, got something else")
+		require.EqualError(t, err, "expecting []byte public key")
 		require.Nil(t, resolvedPubKey)
 	})
 }
+
+type dummyKeyResolver struct {
+	pubKey []byte
+}
+
+func (kr *dummyKeyResolver) Resolve(string) ([]byte, error) {
+	return kr.pubKey, nil
+}
```

Comment:
Should be cheaper and simpler e.g usage `dummyKeyResolver(bytes)`
```
type dummyKeyResolver []byte

func (kr dummyKeyResolver) Resolve(string) ([]byte, error) {
	return kr, nil
}
```

Reply:
Yes, I agree with that. On one hand, it's a struct definition in test code. But we need to strive to have test code even better than production one. :)